Ruby Basic/task5.rb

```diff
@@ -0,0 +1,15 @@
+filename = ARGV[0]
+find_string = ARGV[1..-1].join(' ')
+puts "Going to open '#{filename}'"
+
+text = open filename
+begin
+  text.each.with_index(1) do |line, number_of_string|
+    if line.include?(find_string)
```

Comment:
it is better to use here `next` instead `if` statement, to skip iteration unless line include substring

Reply:
Done